### PR TITLE
Add HACBS Enterprise Contract settings

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Fixes 
Fixes [HACBS-1127](https://issues.redhat.com/browse/HACBS-1127)

## Description
Adds Enterprise Contract info to the settings page for HAC BS.

## Type of change
- [x] Feature
- [ ] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 
Updates to getting started info:
image

Package Info:
image

Expanded:
image

Github link:
image

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge

### CODE REVIEW

File: src/hacbs/components/EnterpriseContractView/useEnterpriseContractPolicies.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      if (!canceled) {
+        setContractPolicies(contractInfo);
+      }
+    });
+    return () => {
+      canceled = true;
+    };
+  }, [fileName]);
+
+  return contractPolicies;
```
Comment (christianvogt) [inline src/hacbs/components/EnterpriseContractView/useEnterpriseContractPolicies.tsx:21]: There should be a loading state.
Also useful if in the future we have a different mechanism for fetching this information.

Typical pattern is `[items, loading, error]`

**Comment by jeff-phillips-18** [comment]
Updated list view:
image

/cc @MariaLeonova

**Comment by MariaLeonova** [comment]
Hi @jeff-phillips-18 ! Thank you for working on this design, it looks really good!
Looking at the [updated list view here](https://github.com/openshift/hac-dev/pull/214#issuecomment-1248338680), I think there's still some opportunity to bring the description text even closer to the Rules names.

I've also noticed that there are no headers in this table, do you think we could add them? The design [here](https://docs.google.com/document/d/1OB8LapdXidPEE_au830sJ7iIfd9iP7e3YYVbYFY_v1A/edit#bookmark=id.5mzvastycibw) specifies Rule and Description as table headers.

**Comment by jeff-phillips-18** [comment]
@MariaLeonova To bring them closer would be to guess at the length of the links. We can but let me know how close and if we really want the links to wrap. I'd have to set a fixed width on the columns, currently it is at 50/50.

We don't typically use titles on data lists, but I can add them.

**Comment by jeff-phillips-18** [comment]
Updated with titles and reduced horizontal separation:

image

**Comment by MariaLeonova** [comment]
This looks very good, @jeff-phillips-18 ! Thank you for implementing the changes!

**Pull request merged**